druid/src/tests/harness.rs

```diff
@@ -45,15 +45,15 @@ pub(crate) const DEFAULT_SIZE: Size = Size::new(400., 400.);
 /// if you want those functions run you will need to call them yourself.
 ///
 /// Also, timers don't work.  ¯\_(ツ)_/¯
-pub struct Harness<'a, T> {
+pub struct Harness<'a, T: Data> {
```

Comment:
what's the motivation for the `Data` bound?

Reply:
Oh yeah, that's no really related to the rest of the PR.

Having the Data bound is useful to understand what `Harness` stores at a glance. It's not super important one way or another.

Reply:
I subscribe to the idea that bounds in struct declarations should be as minimal as possible, so I'd prefer to revert this.